### DISCUSSION TITLE
Changing color of points in User Guide

### DISCUSSION THREAD

**Discussion by tkoyama010** [pyvista/pyvista#1848] [Ideas]
We cannot see the points of figure in [Using Python Lists or Tuples](https://dev.pyvista.org/user-guide/data_model.html#using-python-lists-or-tuples). It is because the color of point and background is same. It is better to change the color of point to red.
![data_model-1_00_00](https://user-images.githubusercontent.com/7513610/143205644-204442b1-8c87-4020-aa89-9f04960637fc.png)

**Comment by banesullivan** [comment]
The default color for the "document" theme should be `'tan'`. The question is why isn't that being used in that example
